tool/randomizer.lua

```diff
@@ -90,31 +92,37 @@ function randomizer.cyclenewgamenewseed()
 end
 
 function randomizer.getdifficulty()
-    return queue[2].difficulty
+    return difficulty
 end
 
 function randomizer.cycledifficulty()
-    local index = table.indexof(difficulties, queue[2].difficulty)
+    local index = table.indexof(difficulties, difficulty)
     index = ((index - 1 + 1) % #difficulties) + 1
-    queue[2].difficulty = difficulties[index]
+    difficulty = difficulties[index]
 end
 
 function randomizer.setnextseed(seedtype, seed)
```

Comment:
Probably fitting to rename it to `addnext` as it isn't only a seed and it doesn't set it anymore.